03.asynchronous/callback.js

```diff
@@ -0,0 +1,60 @@
+import sqlite3 from "sqlite3";
+import timers from "timers/promises";
+
+//エラーなし
+const db = new sqlite3.Database(":memory:");
+
+db.run("DROP TABLE IF EXISTS books", () => {
+  db.run(
+    "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, title VARCHAR(10) NOT NULL UNIQUE)",
+    () => {
+      db.run('INSERT INTO books(title) VALUES("チェリー本")', function () {
```

Comment:
SQL のスタイルが他の部分と揃っていなさそうです。16 行目についても同様です。

```suggestion
      db.run('INSERT INTO books (title) VALUES ("チェリー本")', function () {
```

また、標準 SQL の文字列リテラルは、ダブルクォートではなくシングルクォートです。

Reply:
SQLの文字列はシングルクォートなのですね。どこかで勉強しているはずですが、忘れてしまっていました💦
スタイルを統一するよう修正しました。https://github.com/SuzukaHori/js-practices/pull/3/commits/20290a8031219985b62904e9cc988162f52fe6f8